Showcase-iOS/Contact Us/View/ContactUsViewController.swift

```diff
@@ -0,0 +1,99 @@
+
+
+import Foundation
+import UIKit
+
+class ContactUsViewController: UIViewController {
+    
+    //MARK: Properties
+    
+    var contactUsPresenter: ContactUsPresenter?
+    var cellViewModel: ContactUsCellViewModel?
+    var officeViewModels = [OfficeViewModel]()
+    
+    //MARK: @IBOutlets
+    
+    @IBOutlet weak var collectionView: UICollectionView!
+    
+    //MARK: Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationController?.navigationBar.topItem?.title = "Contact Us"
+        setCollectionView()
+        registerContactUsCell()
+        contactUsPresenter?.retrieveContacts()
+    }
+    
+    //MARK: Operations
+    
+    func setCollectionView() {
+        collectionView.dataSource = self
+        collectionView.delegate = self
+    }
+    
+    func registerContactUsCell() {
+        let contactUsNib  = UINib(nibName: ContactUsCollectionViewCell.identifier, bundle: nil)
+        collectionView.register(contactUsNib, forCellWithReuseIdentifier: ContactUsCollectionViewCell.identifier)
+    }
+    
+    //MARK: MockData
+    
+    func mockValidOfficeResponse() -> OfficeViewModel {
+        let mockJhbOffice: [String: Any] = ["latitude":"-26.1159126", "name":"Johannesburg","image":"offices/dvt_hyde_park.png",
+                                            "googleMapsPlaceId":"ChIJF0f-kTdzlR4RioXEaM2-a10",
+                                            "address":"Ground Floor,Victoria Gate South,Hyde Lane Office Park,Hyde Park Lane,Hydepark,Johannesburg,2196",
+                                            "googleMapsName":"DVT Johannesburg",
+                                            "emailAddress":"jvandermerwe@jhb.dvt.co.za",
+                                            "longitude":"28.0328262",
+                                            "telephone":"+2773444000"]
+        return OfficeViewModel(with: Office(with: mockJhbOffice))
+    }
+}
+
+//MARK: CollectionView extension
+
+extension ContactUsViewController: UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout{
+    
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        return CGSize(width: collectionView.frame.width, height: 236)
```

Comment:
magic number? where did you get this number

Reply:
That's the height of the nib.

Reply:
should the height not be determined by the collection view cell? and should be dynamic. If you happy to refactor later. Please make a ticket. But also need to resolve merge conflicts

Reply:
I will refactor later. I'm fixing the conflicts now. Will add a ticket also.